sources/common/include/Options.h

```diff
@@ -20,8 +20,8 @@
  * @brief Options header file
  */
 
-/// @brief Namespace for options library
-namespace options {
+/// @brief Namespace for common components to dynawo-launcher
+namespace common {
```

Comment:
common seems quite generic, maybe we should put DFL somewhere?

Reply:
DFL ?

Reply:
DynaFlow-Launcher